exercises/practice/bob/.docs/instructions.append.md

```diff
@@ -0,0 +1 @@
+Bob always responds with a single sentence, regardless of the number of lines in the input.
```

Comment:
Exercism markdown standard: every markdown file needs a level-1 header.

```suggestion
# ignore

Bob always responds with a single sentence, regardless of the number of lines in the input.
```

Reply:
Will the header be ignored?

Reply:
Yes, but it needs to exist.